landoscript/src/landoscript/script.py

```diff
@@ -98,10 +98,12 @@ async def async_main(context):
                     if version_bump_action:
                         lando_actions.append(version_bump_action)
                 elif action == "tag":
-                    tag_actions = tag.run(payload["tags"])
+                    tag_actions = await tag.run(session, tag.HgTagInfo(**payload["tag_info"]))
```

Comment:
nit: explicitly check that tag_info has a hg_repo_url?

Reply:
Sure; we'll error out anyways due to the dataclass constructor, but it's probably better to have a TaskVerificationError than a TypeError.